netmiko/dnos/dnos_ssh.py

```diff
@@ -187,11 +188,15 @@ def find_prompt(
         else:
             return prompt
 
-    def _enter_shell(self) -> str:
-        """Enter the Bourne Shell of the routing-engine container on the
-        active NCC"""
+    def _enter_shell(self,ncm=None, ncc="active") -> str:
```

Comment:
Can you do an input check to make sure the NCM and NCC values are verified before going to the cmd stage. Sorry missed it in the first PR.